tests/object/tree/read.c

```diff
@@ -79,46 +79,34 @@ void test_object_tree_read__two(void)
 
 void test_object_tree_read__largefile(void)
 {
-	git_reference *ref;
+	const git_tree_entry *entry;
+	git_index_entry ie = { 0 };
```

Comment:
CI [NAKed](https://dev.azure.com/libgit2/libgit2/_build/results?buildId=2095&view=logs&jobId=7351e2b3-4d1a-54ac-0ff4-44bd85f4818d&taskId=49622b32-88bd-5294-a7a5-08280f44966c&lineStart=602&lineEnd=603&colStart=1&colEnd=1) this.